decidim-admin/app/views/layouts/decidim/admin/_application.html.erb

```diff
@@ -13,7 +13,7 @@
     <%= render partial: "layouts/decidim/admin/header" %>
   </head>
   <body>
-    <%= render partial: "layouts/decidim/timeout_modal" %>
+    <%= render partial: "layouts/decidim/admin_timeout_modal" %>
```

Comment:
If this is a partial that is being used only by admin, maybe it has more sense to be stored in any of the following paths:
- `decidim-admin/app/views/layouts/decidim/admin`
- `decidim-admin/app/views/decidim/admin/shared`

Reply:
Moved